### ISSUE TITLE
fail if output has failed

### ISSUE THREAD

**Issue by yarikoptic** [neurolabusc/dcm2niix#53]
cont to #51 

```shell
$> dcm2niix -f '%s-%d' -o niis ../../bids_* 
Chris Rorden's dcm2niiX version 29Sept2016 (64-bit Linux)
Error: output folder invalid: niis
Error: output folder invalid: niis
Error: output folder invalid: niis
Error: output folder invalid: niis
Error: output folder invalid: niis
Error: output folder invalid: niis
Error: output folder invalid: niis
Error: output folder invalid: niis
Error: output folder invalid: niis
Error: output folder invalid: niis
Error: output folder invalid: niis
Conversion required 0.000052 seconds.

$> echo $?
0
```

IMHO now I would need to capture/parse output to figure out if conversion succeeded or not.  In spirit of "least surprise"  and other tools behavior, imho it would be better if dcm2niix exited with non-0 exit in case of any such grave error.  You can even allocate a few different exit codes for different abnormal operations, e.g.

3 -- output folder is invalid
4 -- ...

;)
Also not sure why to drag further if that output folder was invalid, it will stay invalid for other sequences -- why not to exit right away?

**Comment by neurolabusc** [comment]
Now terminates on first failure and returns EXIT_FAILURE. I agree we could come up with various error codes - too much on my plate to implement this now.

**Comment by yarikoptic** [comment]
great -- thank you @neurolabusc .  FWIW -- you might like to make use of ability to annotate commits so they close issues automagically while leaving a connection on what change has fixed a commit.
In short -- you could've added to your commit msg "Closes #53" which would have closed this issue whenever commit appeared in master.  More info:  https://help.github.com/articles/closing-issues-via-commit-messages/
